### PR TITLE
[WIP]: Adding pease park

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

File: knex/seeds/splash-pads.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  { name: 'metz', latitude: 30.252258120681006, longitude: -97.71810034083994 },
+  { name: 'mueller-branch-park', latitude: 30.300190, longitude: -97.702812 },
+  { name: 'ricky-guerrero', latitude: 30.247392178325605, longitude: -97.76433234845857 },
+  { name: 'rosewood', latitude: 30.27109617907063, longitude: -97.71390894361079 },
+  { name: 'pease', latitude: 30.281754288606752, longitude: -97.7521610530982 }]
+
+  await Promise.all(parks.map(async (park) => {
+    const parkExists = await knex.select('*').from('parks').where({ name: park.name })
+    if (parkExists.length === 0) {
+      knex('parks').insert(park);
```
Comment (kiranjthomas) [inline knex/seeds/splash-pads.js]: doesn't this statement need an await?

File: knex/package.json [hunk 2]
```diff
@@ -13,4 +13,4 @@
     "knex": "^0.95.6",
     "pg": "^8.6.0"
   }
-}
+}
```
Comment (kiranjthomas) [inline knex/package.json]: stray space?

**Review by kiranjthomas — APPROVED** [review]
LGTM; I'd be ok with using a migration to add Pease park which would reduce the change set. Up to you!

File: knex/seeds/splash-pads.js (hunk 1 above)
Comment (geramirez) [reply knex/seeds/splash-pads.js]: It did, but I updated it to use the migrations.

**Pull request merged**